<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QKeyClass</class>
 <widget class="QWidget" name="QKeyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QKey</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>MAC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="macEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>CPU</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cpuEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>timer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="timeEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="keyEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="keyButton">
         <property name="text">
          <string>Key</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qkey.qrc"/>
 </resources>
 <connections/>
</ui>
